cfme/cloud/keypairs.py

```diff
@@ -184,6 +185,41 @@ def download_private_key(self):
         view.toolbar.configuration.item_select('Download private key')
         view.flash.assert_no_error()
 
+    def set_ownership(self, owner=None, group=None, click_cancel=False, click_reset=False):
+        """Set instance ownership
+
+        Args:
+            user (User): user object for ownership
+            group (Group): group object for ownership
+            click_cancel (bool): Whether to cancel form submission
+            click_reset (bool): Whether to reset form after filling
+        """
+        view = navigate_to(self, 'SetOwnership', wait_for_view=0)
+        fill_result = view.form.fill({
+            'user_name': owner if owner else None,
+            'group_name': group if group else None})
+        if not fill_result:
+            view.form.cancel_button.click()
+            view = self.create_view(navigator.get_class(self, 'Details').VIEW)
+            view.flash.assert_success_message('Set Ownership was cancelled by the user')
```

Comment:
_Question_ do you think this should raise some sort of exception?

Reply:
**Required**

only `flash.assert_no_error` should be used here, and the specific flash message text should be asserted against within the CRUD-type test.

This way a change to the text of the flash message on success doesn't block all test cases using this method.

Reply:
Done.

Reply:
@nachandr I am not seeing the change that @mshriver requested about `view.flash.assert_no_error`

Reply:
@john-dupuy , I moved the specifoc flash message text to the test , but forgot to add the view.flash.assert_no_error message. Fixed it.

Reply:
Thanks!